capstone/src/test/resources/com/google/univiz/scorecard/ScorecardTest.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.scorecard;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class ScorecardTest {
+
+  @Test
+  public void parseJsonTest() {
+    // TODO(biancamacias): insert sample JSON
+    String json = "{}";
+    ScorecardData college = new Gson().fromJson(json, ScorecardData.class);
+
+    asserThat(college.name().isEqualTo("New York University"));
+    asserThat(college.city().isEqualTo("New York City"));
```

Comment:
I don't believe this PR will build. Can you sync past #47 so that the Java builder check runs on this PR?